hardinfo.1

```diff
@@ -0,0 +1,58 @@
+.\" Manpage for hardinfo.
+.\" https://github.com/lpereira/hardinfo/.
+.TH HARDINFO "1" "July 2017" "0.6" "User Commands"
+.SH NAME
+Hardinfo \- System profiler and benchmark for Linux systems.
+.SH USAGE
+hardinfo [OPTION...]
+.SH DESCRIPTION
+HardInfo is a system profiler and benchmark for Linux systems.
+
+HardInfo can gather information about your operating system 
+and detected hardware by them, perform benchmarks, and then 
+generate printable reports either in HTML or in plain text formats.
+
+
+Features include:
+ *\  System information, like kernel, version, desktop, users, network
```

Comment:
This list isn't adding much information on top of what's written above. I'd remove it.